### DISCUSSION TITLE
Allow song directory input when importing from stable

### DISCUSSION THREAD

**Discussion by NoelleTGS** [ppy/osu#26861] [Ideas] [answered]
I've ran stable through Wine on Linux for a while now, and when going through lazer's first-time setup importer, importing songs gives an error. I assume this is because it tries to read my song location in the .cfg, and since it's a Windows directory (manually mapped through the Wine config), Linux/Lazer has no idea where "D:\Songs" is.

Letting users manually enter a directory to import from would solve this issue, though I do get that it's a stretch considering how few users will encounter this issue.

**Comment by peppy** [comment, marked as answer]
You should be able to manually select the songs folder already.

If it's not letting you, please delete or remove the .cfg temporarily.

**Reply by NoelleTGS** [reply]
![image](https://github.com/ppy/osu/assets/44215890/91152ff8-f95e-46a1-b376-ee410a36c624)
![image](https://github.com/ppy/osu/assets/44215890/a307ced6-8520-460a-abc0-876130e4d30a)
I'm not seeing an option even after removing the .cfg file.

**Reply by NoelleTGS** [reply]
Ahhh okay nevermind, I see what you mean. Had to first import by selecting the drive containing the songs, then import the main installation.
